Formula/unit.rb

```diff
@@ -0,0 +1,95 @@
+class Unit < Formula
+  desc "Dynamic web and application server for multiple languages"
+  homepage "https://unit.nginx.org/"
+  url "https://unit.nginx.org/download/unit-1.1.tar.gz"
+  sha256 "7c66365f5ea87e7e8903bac9d639676947fe2ab7aa799ce35defc426b3409ee0"
+  head "https://hg.nginx.org/unit", :using => :hg
+
+  option "with-debug", "Build with debug logging enabled"
+
+  depends_on "openssl"
+  depends_on "pcre"
+
+  depends_on "go" => :optional
+  depends_on "perl" => :optional
+  depends_on "python" => :optional
+  depends_on "ruby" => :optional
+
+  def install
+    openssl = Formula["openssl"]
+    pcre = Formula["pcre"]
+
+    configure_args = [
+      "--prefix=#{prefix}",
+      "--bindir=#{bin}",
+      "--sbindir=#{bin}",
+      "--modules=#{lib}/unit",
+      "--state=#{var}/unit",
+      "--log=#{var}/log/unit.log",
+      "--pid=#{var}/run/unit.pid",
+      "--control=unix:#{var}/run/unit.control.sock",
+      "--cc-opt=-I#{pcre.opt_include} -I#{openssl.opt_include}",
+      "--ld-opt=-L#{pcre.opt_lib} -L#{openssl.opt_lib}",
+    ]
+
+    if build.with? "debug"
+      configure_args << "--debug"
+    end
+
+    system "./configure", *configure_args
```

Comment:
No extra variable `configure_args`, inline all arguments to configure here.

Reply:
I started that way, then extracted them when "--debug" was optional. Without that option, no wuckers. 👌